src/SearchPage.js

```diff
@@ -62,17 +62,18 @@ export default class SearchPage extends Component {
                 <h1> Find Your Pokemon</h1>
                 <p> Search for Pokemon or filter using dropdown below</p>
                 <p> Page Number: {this.state.currentPage}</p>
-                <form onSubmit={this.submitPokemon}>
-                    <select onChange={this.pokemonSort}>
-                        <option value="asc"> ASC</option>
-                        <option value='desc'>DSC</option>
-                    </select>
-                    <input onChange={this.findPokemon} />
-                    <button>Search</button>
-                </form>
-                {this.state.currentPage !== 1 ? <button onClick={this.previousPageClick}> Prev </button> : null}
-
-                <button onClick={this.nextPageClick}> Next </button>
+                {this.state.loading ? <img src="https://www.supereasy.com/wp-content/uploads/2020/02/2020-02-12_19-13-36.jpg" alt="" /> :
```

Comment:
the other option for the loading state should be the PokeList.
Move lines 66-73, and lines 75-76 to above line 65. 
Then move line 77 to line 66.